src/App.js

```diff
@@ -1,24 +1,27 @@
-import logo from './logo.svg';
+import React from 'react';
+import {
+  BrowserRouter as Router, Routes, Route, Link,
+}
+  from 'react-router-dom';
 import './App.css';
+import Categories from './components/Categories';
+import Book from './components/BookList';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <nav>
+        <ul>
+          <li><Link to="/">Home</Link></li>
+          <li><Link to="/categories">Categories</Link></li>
+        </ul>
+      </nav>
+
```

Comment:
- [ ] Please from the project requirement, Inside of the `/components` directory: You should create a component for this nav which contains navigation to link to the 2 routes you created :pray:  I suggest that you create a separate component for this navigation and render the component your app component. :blush:

Reply:
well resolved! ✔️